libraries/app/include/graphene/app/database_api.hpp

```diff
@@ -667,6 +667,14 @@ class database_api
        */
       vector<tournament_id_type> get_registered_tournaments(account_id_type account_filter, uint32_t limit) const;
 
+      //////////
+      // RNG //
+      //////////
+      /**
+       * @return Current Random number
```

Comment:
I think we can add the `bound` argument with a brief description above here. Also maybe change `RNG` with `Random Number Generator`, is a bit confusing on the first sight.